kura/test/org.eclipse.kura.net.configuration.test/.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
What is the purpose of this file?

Reply:
This a spourius file. I'll update the .gitignore file to ignore it.